docs/4.0/i18n/zh-Hans/Development-Guidelines/resource-controller.md

```diff
@@ -0,0 +1,119 @@
+# Resource-controller如何编写
+
+resource-controller 是计量计费模块的一部分，请确保您提前了解[计量计费的设计](https://github.com/labring/sealos/blob/main/proposal/design/zh/metering%E8%AE%BE%E8%AE%A1.md)
```

Comment:
use markdown link to relative url like [](/proposal/design/zh/metering%E8%AE%BE%E8%AE%A1.md)

Reply:
linke this ? ](../../../../../proposal/design/zh/metering%E8%AE%BE%E8%AE%A1.md)

Reply:
> linke this ? (../../../../../proposal/design/zh/metering%E8%AE%BE%E8%AE%A1.md)

Just `/proposal/design/zh/metering%E8%AE%BE%E8%AE%A1.md` is enough?